doc/installation/system/inifile.rst

```diff
@@ -102,6 +102,11 @@ can be set, for example::
 Further information on possible parameters can be found in the
 `PassLib documentation <https://passlib.readthedocs.io/en/stable/lib/passlib.hash.html>`_.
 
+In ``PI_HSM_PADDING`` the padding can be set to "PKCS7". If ``PI_HSM_PADDING`` is not set, the default internal
+padding is used::
```

Comment:
privacyIDEA currently has three security Modules:

* DefaultSecurityModule
* AESHardwareSecurityModule
* EncryptKeyHardwareSecurityModule (new)

Please note, this is only valid for the `DefaultSecurityModule` and the `EncryptKeyHardwareSecurityModule`.

The `AESHardwareSecurityModule` will always use CKM_AES_CBC_PAD (which is PKCS7, in my understanding).
So this config value is only functional for the above two mentioned two Security Modules. 

Which is fine, but which I think we should mention here.